### PR TITLE
Adjusted config procedures to be consistent

### PR DESCRIPTION
**Pull request by TinaUll**

### CODE REVIEW

File: V1.2/Configuration/pi-adapter-for-opc-ua-configuration-examples.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+            },
+            {
+                "id": "OCS Endpoint",
+                "endpoint": "https://OCSEndpoint/omf",
+                "clientId": "CLIENTID",
+                "clientSecret": "CLIENTSECRET"
+            },
+            {
+                "Id": "EDS",
+                "Endpoint": "http://localhost:/api/v1/tenants/default/namespaces/default/omf"
```
Comment (osisoft-mbishop) [inline V1.2/Configuration/pi-adapter-for-opc-ua-configuration-examples.md:107]: missing a comma here that will resolve the linting on the next line.

```suggestion
                "Endpoint": "http://localhost:/api/v1/tenants/default/namespaces/default/omf",
```

File: V1/Configuration/pi-adapter-for-opc-ua-configuration-examples.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+            },
+            {
+                "id": "OCS Endpoint",
+                "endpoint": "https://OCSEndpoint/omf",
+                "clientId": "CLIENTID",
+                "clientSecret": "CLIENTSECRET"
+            },
+            {
+                "Id": "EDS",
+                "Endpoint": "http://localhost:/api/v1/tenants/default/namespaces/default/omf"
```
Comment (osisoft-mbishop) [inline V1/Configuration/pi-adapter-for-opc-ua-configuration-examples.md:107]: Same comma issue

```suggestion
                "Endpoint": "http://localhost:/api/v1/tenants/default/namespaces/default/omf",
```

**Review by osisoft-mbishop — APPROVED** [review]
A couple of the full config examples are missing a comma, but I have code suggestions upthread to fix them. Fix those and you're good to merge. 👍

**Comment by TinaUll** [comment]
Thanks @osisoft-mbishop. Good catch!

**Pull request merged**